src/node_crypto.cc

```diff
@@ -3952,6 +3954,37 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment* env, v8::Local<v8::Object> options,
```

Comment:
But doesn't that imply passing the padding etc. from JavaScript to the binding as additional arguments instead of an `options` object? I considered that as well, but assuming that RSASSA-PSS is not the last addition to `sign()`/`verify()`, it decided it could become pretty clumsy to pass all options as positional parameters.